### ISSUE TITLE
Is DLTbackbone model ready to be used?

### ISSUE THREAD

**Issue by drixs2050** [JulianGuerreiro/LayoutFlow#3]
Hello there, 
Really appreciate the comprehensive code you implemented, there are 2 things i want to confirm:
Is the layoutdmx you included the same as the the original layoutdm? If not what is the change made here?
I see that included DLT as in backbone model as well, is it the same as the original DLT model and is it ready to be used? If yes, can I simply use it by changing backbone model in config file? is there anything else I need to change?
Thank you very much!

**Comment by JulianGuerreiro** [comment]
Hi,
thank you for your message!

Both LayoutDM and DLT backbones are *not* the original models but use the same or a very similar transformer architecture as these methods. For the results in our paper, we used the LayoutDM backbone architecture but the results we got using the DLT backbone were actually quite similar and only slightly worse. To be clear here, the training procedure and other settings are quite different from DLT and LayoutDM, this is only the architecture blocks of the models that are similar. 

Regarding `LayoutDMx`, I'm sorry the name is a little unfortunate here. `LayoutDMx` has not a lot to do with LayoutDM (which uses *discrete* diffusion). `LayoutDMx` implements *continuous* diffusion and we implemented it to directly compare *continuous* diffusion with *continuous* flow matching. 

In summary, `LayoutDMx` and `LayoutFlow` describe the training procedure (diffusion and flow matching, respectively), while the backbones only define the transformer architecture that is used for each denoising step.

**Comment by drixs2050** [comment]
I see, so just want to confirm are the results in your paper obtained using the original implementation of DLT and layoutDM (also layoutdiffusion)?

**Comment by JulianGuerreiro** [comment]
Yes, exactly! We use the original implementation, but had to retrain DLT and LayoutDM using the dataset split in LayoutDiffusion.
